turborepo-tests/helpers/setup_example_test.sh

```diff
@@ -55,24 +55,28 @@ mkdir -p ./tmp
 echo "/tmp/" >> ".gitignore"
 
 # Simulating the user's first run and dumping logs to a file
+$package_manager --version >./tmp/package_manager_version.txt
 $package_manager_command >./tmp/install.txt 2>&1
-$turbo_command >./tmp/grep-me-for-miss.txt
+$turbo_command >./tmp/run-1.txt
 
 # We don't want to hit cache on first run because we're acting like a user.
 # A user would never hit cache on first run. Why should we?
-if grep -q ">>> FULL TURBO" ./tmp/grep-me-for-miss.txt; then
-  echo "A FULL TURBO was found. This test is misconfigured (since it can hit a cache)."
+if grep -q ">>> FULL TURBO" ./tmp/run-1.txt; then
+  echo "[ERROR] A 'FULL TURBO' was found. This test must be misconfigured since it hit a cache on what was expected to be the very first run."
   echo "Dumping logs:"
-  cat ./tmp/grep-me-for-miss.txt >&2
+  echo ""
+  cat ./tmp/run-1.txt >&2
   exit 1
 fi
 
 # Simulating the user's second run
-$turbo_command >./tmp/grep-me-for-hit.txt
+$turbo_command >./tmp/run-2.txt
 
 # Make sure the user hits FULL TURBO on the second go
-if ! grep -q ">>> FULL TURBO" ./tmp/grep-me-for-hit.txt; then
-  echo "No FULL TURBO was found. Dumping logs:"
-  cat ./tmp/grep-me-for-hit.txt >&2
+if ! grep -q ">>> FULL TURBO" ./tmp/run-2.txt; then
+  echo "[ERROR] No 'FULL TURBO' was found.  That shows that the cache was not fully hit on the second run, which was expected behavior."
```

Comment:
At a glance this phrasing reads like the second run cache miss is _expected_. Maybe just re-word to something like:


```suggestion
  echo "[ERROR] No 'FULL TURBO' was found.  That indicates that the second run unexpectedly missed the cache."
```

Reply:
good feedback: updated in 5cd348278bb51744c3afeac32898e6690d34e061 to make it clear that "hitting the cache" isn't the same as seeing a wall full of "cache hit" as long as there's even one "cache miss".